modules/network/src/integrationTest/java/org/apache/ignite/internal/network/scalecube/ItMulticastNodeFinderTest.java

```diff
@@ -67,34 +70,46 @@ void setUp(TestInfo testInfo) {
     @AfterEach
     void tearDown() {
         assertThat(stopAsync(new ComponentContext(), services), willCompleteSuccessfully());
+        services.clear();
+        finders.forEach(NodeFinder::close);
```

Comment:
Let's use `IgniteUtils.closeAll()` to close as many 'things' we can, even if some of them throw an exception.